docs/framework/data/adonet/sql/linq/getting-started.md

```diff
@@ -25,5 +25,5 @@ By using [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)], yo
   
 ## See also
 - [LINQ to SQL](../../../../../../docs/framework/data/adonet/sql/linq/index.md)
-- [Introduction to LINQ](https://msdn.microsoft.com/library/24dddf19-12a0-4707-a4bc-eba4fa7f219e)
+- [Introduction to LINQ](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/bb397897(v=vs.120))
```

Comment:
Split these in two:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/introduction-to-linq
https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/linq/introduction-to-linq